src/cmd-kola

```diff
@@ -56,6 +56,8 @@ kolaargs = ['kola']
 if os.getuid() != 0 and not ('-p' in unknown_args):
     kolaargs.extend(['-p', 'qemu-unpriv'])
 
+if args.build is not None:
+    kolaargs.extend(['--cosa-build', args.build])
```

Comment:
Why not just call the argument in kola `--build`?  Then we could drop this right?

Reply:
I did this (and actually went one step further renaming `--cosa-workdir` to just `--workdir`) because yeah, kola now lives in cosa and we should reflect that.

Re. dropping the `--build` dir, I left that alone for now because the passthrough unknown args hack falls apart for switches which take values. It'll all be resolved anyway once `cosa kola "$@"` is just an `exec kola "$@"`.